tests/expr_and_series/name/map_test.py

```diff
@@ -20,29 +14,20 @@ def map_func(s: str | None) -> str:
 def test_map(constructor: Constructor) -> None:
     df = nw.from_native(constructor(data))
     result = df.select((nw.col("foo", "BAR") * 2).name.map(function=map_func))
-    expected = {map_func(k): [e * 2 for e in v] for k, v in data.items()}
+    expected = {"oof": [2, 4, 6], "rab": [8, 10, 12]}
```

Comment:
@FBruzzesi for the future, i'd much prefer to constructor expected output explicitly (no blame of course! just a consideration for future developments)

Reply:
Of course! I think this is quite old (with respect to narwhals development), after which we started to always explicitly build the expected dictionary 👌